dataclients/kubernetes/kube.go

```diff
@@ -969,12 +896,12 @@ func (c *Client) ingressToRoutes(items []*ingressItem) ([]*eskip.Route, error) {
 			// update Traffic field for each backend
 			computeBackendWeights(backendWeights, rule)
 
-			for idx, prule := range rule.Http.Paths {
+			for _, prule := range rule.Http.Paths {
 				// add extra routes from optional annotation
 				for extraIndex, r := range extraRoutes {
 					route := *r
 					route.HostRegexps = host
-					route.Id = routeIDForCustom(i.Metadata.Namespace, i.Metadata.Name, route.Id, rule.Host+strings.Replace(prule.Path, "/", "_", -1), idx, extraIndex)
```

Comment:
i think we cannot drop the index of the path rule from the generated id of the custom route, because we make a copy of the custom routes for every path rule.

Reply:
i get it: the path itself will make it unique

Reply:
I think we should move this change to another PR